lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/table/QueryFacetResultRange.json

```diff
@@ -0,0 +1,21 @@
+{
+	"description": "Describes the value range selected by the user and the absolute minimum and maximum of a column's values",
```

Comment:
Includes the minimum and maximum values of a range facet column and selected ranges applied to the filter.